src/components/ListItem.tsx

```diff
@@ -80,6 +81,26 @@ export function ListItemCheckBox({ item, listPath }: Props) {
 		}
 	};
 
+	const editItemQuantity = async (quantity: number) => {
```

Comment:
More a comment for later if we ever do a refactor. It may still be beneficial to move the logic for updating the item quantity to be contained in the `ItemQuanitiyForm`

Nonblocking!

Awesome functionality to add to the app!